pkg/storagev2/migration/migration_test.go

```diff
@@ -118,12 +194,22 @@ func TestMigrate(t *testing.T) {
 			t.Errorf("GetVersion() error = %v", err)
 		}
 		if newVersion != 8 {
-			t.Errorf("new version = %v must be 3", newVersion)
+			t.Errorf("new version = %v must be 8", newVersion)
 		}
 
-		// TODO Assert that migrated values are in store
+		ObjectExists(t, s, 3, objT1, objT2, objT3)
 	})
 }
+func ObjectExists(t *testing.T, s storage.Store, expectedKeys int, keys ...storage.Key) {
+	if len(keys) != expectedKeys {
+		t.Errorf("expected %v keys, got %v", expectedKeys, len(keys))
+	}
```

Comment:
`expectedKeys` is passed in argument which checks passed in arguments? Should it be fine to trust passed in `keys`?

Reply:
Usually test helpers functions like `ObjectExists` start with prefix `assert` eg.  `assertObjectExists`.